com.synology.synology-note-station.metainfo.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.synology.synology-note-station</id>
```

Comment:
this already exist https://github.com/flathub/com.synology.synology-note-station